packages/bsky/src/hydration/hydrator.ts

```diff
@@ -127,13 +127,27 @@ export class Hydrator {
   async hydrateProfileViewers(
     dids: string[],
     ctx: HydrateCtx,
+    hydrateKnownFollowers = false,
```

Comment:
mer feel like this param is a little out of place in this class... we don't have any other methods with parameter switches like this. but also not really seeing how to get around it

Reply:
well actually, combining this with the feedback from actor hydrator, one option would be just to treat `profileKnownFollowers` as a separate hydration map entirely on `HydrationState` (alongside `profileViewers` instead of inside it). Then we wouldn't need to parameterize this method, it could clean up the actor hydrator a bit, and we would just call that function from within `hydrateProfilesDetailed` 👀

Reply:
does that make any sense? sorry it's a bit bigger of a change, but should just be moving logic around